x/supply/types/permissions.go

```diff
@@ -0,0 +1,20 @@
+package types
+
+import "errors"
+
+// permissions
+const (
+	Holder = "holder"
+	Minter = "minter"
+	Burner = "burner"
+)
+
+// validate the input permission
+func validatePermission(permission string) error {
```

Comment:
again, too rigid, but for a follow up PR.

Reply:
This could be validate permissions

Reply:
updated